src/app/(profile)/profile-edit/page.tsx

```diff
@@ -97,121 +107,251 @@ const UserProfileForm: React.FC = () => {
                   <input
-                    type='file'
+                    type="file"
                     ref={fileInputRef}
-                    className='hidden'
+                    className="hidden"
                     onChange={handleImageChange}
-                    accept='image/*'
+                    accept="image/*"
                   />
-                  <div 
-                    className='absolute bottom-0 right-0 bg-blue-500 rounded-full p-1.5 cursor-pointer'
+                  <div
+                    className="absolute bottom-0 right-0 bg-blue-500 rounded-full p-1.5 cursor-pointer"
                     style={{ transform: 'translate(25%, 25%)' }}
                     onClick={handleImageClick}
                   >
-                    <svg className='w-4 h-4 text-white' fill='none' stroke='currentColor' viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'>
-                      <path strokeLinecap='round' strokeLinejoin='round' strokeWidth={2} d='M15.232 5.232l3.536 3.536m-2.036-5.036a2.5 2.5 0 113.536 3.536L6.5 21.036H3v-3.572L16.732 3.732z' />
+                    <svg
+                      className="w-4 h-4 text-white"
+                      fill="none"
+                      stroke="currentColor"
+                      viewBox="0 0 24 24"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        strokeLinecap="round"
+                        strokeLinejoin="round"
+                        strokeWidth={2}
+                        d="M15.232 5.232l3.536 3.536m-2.036-5.036a2.5 2.5 0 113.536 3.536L6.5 21.036H3v-3.572L16.732 3.732z"
+                      />
                     </svg>
                   </div>
                 </div>
-                <h2 className='text-xl font-semibold text-blue-600 text-center md:text-left'>{`${watchedValues.firstName || ''} ${watchedValues.lastName || ''}`}</h2>
-                <p className='text-sm text-gray-500 mb-4 text-center md:text-left'>{user?.User?.Role.name || 'buyer'}</p>
+                <h2 className="text-xl font-semibold text-blue-600 text-center md:text-left">{`${watchedValues.firstName || ''} ${watchedValues.lastName || ''}`}</h2>
+                <p className="text-sm text-gray-500 mb-4 text-center md:text-left">
+                  {user?.User?.Role.name || 'buyer'}
+                </p>
               </div>
 
-              <ul className='hidden md:block text-sm space-y-2'>
-                <li className='flex items-center text-gray-600'>
-                  <svg className='w-4 h-4 mr-2' fill='none' stroke='currentColor' viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'><path strokeLinecap='round' strokeLinejoin='round' strokeWidth={2} d='M8 7V3m8 4V3m-9 8h10M5 21h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v12a2 2 0 002 2z' /></svg>
-                  {watchedValues.birthDate || "YYYY-MM-DD"}
+              <ul className="hidden md:block text-sm space-y-2">
```

Comment:
This file is too long. You are probably not using reusable components. Refactor this component by dividing it into smaller components that perform specific tasks.